### ISSUE TITLE
Distraction free.mode

### ISSUE THREAD

**Issue by taneltm** [slap-editor/slap#123]
Ability to switch between normal and distraction-free mode.

In the distraction-free mode, the file browser, editor gutter and the header would be hidden.

This would allow one to use all the space in a 80x24 terminal for code editing, which is the default terminal size on many systems.

**Comment by secrettriangle** [comment]
Absolutely. You can do this via config, by setting `gutter.width`, `fileBrowser.width`, and `header.height` to `0`, but it would be nice to have a hotkey for this.

**Comment by piranna** [comment]
> but it would be nice to have a hotkey for this.

F11 - fullscreen? :-)
